File: Java/src/main/java/parrot/EuropeanParrot.java
package parrot;

public class EuropeanParrot extends Parrot {

    protected boolean isNailed;

    public EuropeanParrot() {
        super();
    }

    @Override
    public double getSpeed() {
        return getBaseSpeed();
    }

    @Override
    public String getCry() {
        return "Sqoork!";
    }
}

File: Kotlin/src/main/kotlin/parrot/EuropeanParrot.kt
package parrot

class EuropeanParrot : Parrot {
    constructor(numberOfCoconuts: Int, voltage: Double, isNailed: Boolean) :
            super(type = ParrotTypeEnum.EUROPEAN,
                numberOfCoconuts = numberOfCoconuts,
                voltage = voltage,
                isNailed = isNailed
                    )
    override val speed: Double = baseSpeed
    override val cry: String = "Sqoork!"
}
